perf/macro/merge/perf.js

```diff
@@ -11,7 +11,7 @@ var Rx2Merge = document.querySelector('#rx-2-merge');
 var RxNextMerge = document.querySelector('#rx-3-merge');
 
 var Rx2TestObservable = Rx.Observable.range(0, numIterations);
-var RxNextTestObservable = RxNext.Observablerange(0, numIterations);
+var RxNextTestObservable = RxNext.Observable.range(0, numIterations);
```

Comment:
all typo error comes from this, copy-paste without giving enough caution. Sorry again for error, will keep more eye in further.